From 51bd3bc7c6578ae178299d63dc7aadaff65e9ed6 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sat, 21 Dec 2019 05:52:13 +0200
Subject: [PATCH] [SW-0] preboot: gaudi: finally move to sram fixup goya2

While reverting compilation breaked on poiner conversion due to
different size

Change-Id: I7f4b25b4e31bf94820f89e26a8352920308c6abf
---

diff --git a/app/preboot.c b/app/preboot.c
index d8c409d..197674e 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -102,7 +102,7 @@
 
 static void preboot_version(void)
 {
-	u64 offset = (u64) preboot_btl_ver_str;
+	unsigned long offset = (unsigned long) preboot_btl_ver_str;
 	u32 btlver = regmap_readl(mmPSOC_GLOBAL_CONF_SCRATCHPAD_28);
 	int i, btl_ver_sz;
 
